hack/release.sh

```diff
@@ -90,22 +86,23 @@ for platform in "${CRI_TEST_PLATFORMS[@]}"; do
         -o ${output_bin} \
         -ldflags "${GO_LDFLAGS}" \
         ${PROJECT}/cmd/critest
-    if $UPLOAD_GCS; then
-      gsutil cp ${output_bin} gs://${GCS_PATH}/critest-$VERSION-$os-$arch
-      sha1sum ${output_bin} | awk '{print $1}' | gsutil cp - gs://${GCS_PATH}/critest-$VERSION-$os-$arch.sha1sum
-    fi
-    tar zcvf ${CRI_TOOLS_ROOT}/_output/releases/critest-$VERSION-$os-$arch.tar.gz \
+    file ${output_bin}
+    tar zcf "$OUTPUTDIR/critest-$VERSION-$os-$arch.tar.gz" \
         -C ${CRI_TOOLS_ROOT}/_output/bin/$arch-$os \
         ${CRITEST_BIN}
 done
 
+printf "\n## Downloads\n\n"
+printf "| file | sha256 |\n"
+printf "| ---- | ------ |\n"
+
 # Show sha256 for release files
 if [[ "${OSTYPE}" == "darwin"* ]]; then
-  for file in $(ls ${CRI_TOOLS_ROOT}/_output/releases); do
-    echo "$file $(shasum -a 256 ${CRI_TOOLS_ROOT}/_output/releases/$file | awk '{print $1}')"
-  done
+    for file in $(ls "$OUTPUTDIR"); do
+        echo "| $file | $(shasum -a 256 "$OUTPUTDIR/$file" | awk '{print $1}' | tee "$OUTPUTDIR/$file.sha256") |"
```

Comment:
What do you think about writing the full output so verifiers can run `sha256sum -c`? Adding the `sed` will trim out the absolute path so its just `<hash>  $file`

```suggestion
        echo "| $file | $(shasum -a 256 "$OUTPUTDIR/$file" | sed -e "s,$OUTPUTDIR,,"' | tee "$OUTPUTDIR/$file.sha256" | awk '{print $1}' ) |"
```

Reply:
Sure, added it as suggested :+1: